include/libdnf/rpm/package.hpp

```diff
@@ -23,7 +23,6 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include "checksum.hpp"
 #include "reldep_list.hpp"
-#include "sack.hpp"
```

Comment:
Why did you remove the import? The import was required for SolvSack and PackageId.

Reply:
There is indirect include via "reldep_list.hpp". But in this case I agree with you. I will add the include back.